README.md

```diff
@@ -533,3 +533,19 @@ RubyPGExtras.mandelbrot
 ```
 
 This command outputs the Mandelbrot set, calculated through SQL.
+
+### `buffercache_stats`
+
+This command show you the relations buffered in database share buffer, ordered by relation percentage taken in shared buffer. It also shows that how much of the whole relation is buffered.
```

Comment:
Could you expand the description so that one can use the query without referencing other sources?

Reply:
I'll leave this to one to you, documentation is something that I'm not good at. :grimacing: